### PR TITLE
[Issue 28] Create channels for the WildFly 24 releases

### LINKED ISSUE #28: Create channels for the WildFly 24 releases

### PR DESCRIPTION
**Pull request by bstansberry**
Resolves #28

### CODE REVIEW

**Comment by bstansberry** [comment]
@jfdenise Please review

File: pom.xml [hunk 1]
```diff
@@ -182,6 +182,16 @@
                       <version-range>[15.0-alpha,16)</version-range>
                       <version-exclude-regex>.*-redhat-[0-9][0-9][0-9][0-9][0-9]</version-exclude-regex>
                     </channel>
+                    <channel>
+                      <name>16.0</name>
+                      <version-range>[15.0-alpha,15.1)</version-range>
```
Comment (jfdenise) [inline pom.xml:187]: Copy paste, should be 16.

Comment (bstansberry) [reply pom.xml:187]: Sigh; fixing that now. Thanks!

Comment (bstansberry) [reply pom.xml:187]: @jfdenise Fixed

**Review by jfdenise — APPROVED** [review]

**Pull request merged**